src/main/java/org/threadly/litesockets/NoThreadSocketExecuter.java

```diff
@@ -221,8 +225,8 @@ protected void shutdownService() {
       if(selector != null && selector.isOpen()) {
         selector.close();
       }
-    } catch (IOException e) {
-
+    } catch (Exception e) {
+      ExceptionUtils.getExceptionHandler().handleException(e);
```

Comment:
no worries, moved it to the static call.

On Fri, May 8, 2015 at 10:50 PM, Mike Jensen <notifications@github.com>
wrote:

> In src/main/java/org/threadly/litesockets/NoThreadSocketExecuter.java
> <https://github.com/threadly/litesockets/pull/16#discussion_r29989748>:
>
> > @@ -221,8 +225,8 @@ protected void shutdownService() {
> >        if(selector != null && selector.isOpen()) {
> >          selector.close();
> >        }
> > -    } catch (IOException e) {
> > -
> > +    } catch (Exception e) {
> > +      ExceptionUtils.getExceptionHandler().handleException(e);
>
> getExceptionHandler() may return null if none is set.
>
> ExceptionUtils.handleException() is another option (but if no
> ExceptionHandlerInterface is set, and an UncaughExceptionHandler is set, it
> will be called instead...it does prioritize an ExceptionHandlerInterface
> first though).
>
>
> http://threadly.github.io/threadly/javadocs/3.8.0/org/threadly/util/ExceptionUtils.html#handleException-java.lang.Throwable-
>
> Sorry to keep throwing it back at you, but I do think it looks good
> besides this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/threadly/litesockets/pull/16/files#r29989748>.
>